test/jsdom/virtual-console.js

```diff
@@ -21,10 +22,16 @@ var consoleMethods = [
   "warn"
 ];
 
-exports["jsdom.getVirtualConsole returns an instance of EventEmitter"] = function (t) {
+exports["VirtualConsole inherits from EventEmitter"] = function (t) {
```

Comment:
I think it's OK to leave these tests as-is (as part of the don't-test-private-APIs thing).